docs/pages/reference/access-controls/roles.mdx

```diff
@@ -371,10 +371,9 @@ an administrative role. In particular, you should avoid configuring `allow` rule
 that grant `create` and `update` permissions on `token` resources to prevent
 unexpected changes to the configuration or state of your cluster.
 
-## RBAC for sessions
+## RBAC for session recordings
 
 It is possible to further limit access to
-[shared sessions](../../connect-your-client/tsh.mdx) and
 [session recordings](../architecture/session-recording.mdx).
 The examples below illustrate how to restrict session access only for the user
 who created the session.
```

Comment:
The examples below now illustrate more than just limiting access to the session creator.

I'd reframe this section.
- First, introduce the `where` clause and explain that it can be used to restrict access based on user traits and resource metadata.
- Then have a "recipes" section where you show some common examples (restrict to session participants, restrict to certain roles, traits, etc)

Reply:
Done. Do you mind reading it again?

I added the examples before the list of fields so people can easily see examples.

I wonder if we should create a page dedicated to session RBAC but for now, I decided to avoid it.